### PR TITLE
Update minor issues in docs

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update minor issues in docs

### CODE REVIEW

File: docs/installation/slack/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 ---
 
 :::info
-[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and the old Botkube Slack app will be unapproved from the Slack App Directory on the December 21st 2022. Follow the tutorial below to use the new Slack integration.
+[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and removed from the Slack App Directory. Follow the tutorial below to use the new Slack integration.
```
Comment (pkosiec) [inline docs/installation/slack/index.md:8]: This is something we should also apply to older versions 🤔

Comment (mszostok) [reply docs/installation/slack/index.md:8]: hm.. 🤔 I thought to not do that as for <0.17 it was true :D as they were released before 21st Dec 😄 

But I can update that in all places too.

Comment (pkosiec) [reply docs/installation/slack/index.md:8]: Essentially we can replace all occurrences of `will be unapproved from the Slack App Directory on the December 21st 2022` to `was unapproved from the Slack App Directory on the December 21st 2022` as it already happened, so... 😄

File: versioned_docs/version-0.16/installation/slack/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 ---
 
 :::info
-[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and the old Botkube Slack app will be unapproved from the Slack App Directory on the December 21st 2022. Follow the tutorial below to use the new Slack integration.
+[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and removed from the Slack App Directory. Follow the tutorial below to use the new Slack integration.
```
Comment (pkosiec) [inline versioned_docs/version-0.16/installation/slack/index.md:8]: Please also change that in 0.13, 0.14 and 0.15. Thank you!

BTW We meed to think about when to remove docs for the "archived" versions at some point 🙂

**Review by pkosiec — APPROVED** [review]

File: versioned_docs/version-0.16/installation/slack/index.md (hunk 2 above)
Comment (mszostok) [reply versioned_docs/version-0.16/installation/slack/index.md:8]: ah.. missed that :( Will fix in next commit 👍

**Pull request merged**